RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
 18.7
 -----
 
+18.6.1
+-----
+* [*] Block editor: Fixed a crash that could occur when copying lists from Microsoft Word [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4174]
+
```

Comment:
Note that we agreed to remove these in favor of keeping the fix notes in 18.7 in #17483.